lib/experimental/fonts/font-library/README.md

```diff
@@ -0,0 +1,250 @@
+}
+```
+
+Example of a data JSON file providing 2 font families with 2 font faces each and 2 categories:
+
+```json
+{
+  "fontFamilies": [
+    {
+      "name": "Montserrat",
+      "fontFamily": "Montserrat, sans-serif",
+      "slug": "montserrat",
+      "category": "sans-serif",
+      "fontFace": [
+        {
+          "src": "http://fonts.gstatic.com/s/montserrat/v25/JTUHjIg1_i6t8kCHKm4532VJOt5-QNFgpCtr6Ew-Y3tcoqK5.ttf",
+          "fontWeight": "400",
+          "fontStyle": "normal",
+          "fontFamily": "Montserrat"
+        },
+        {
+          "src": "http://fonts.gstatic.com/s/montserrat/v25/JTUFjIg1_i6t8kCHKm459Wx7xQYXK0vOoz6jq6R9aX9-p7K5ILg.ttf",
+          "fontWeight": "400",
+          "fontStyle": "italic",
+          "fontFamily": "Montserrat"
+        },
+      ]
+    },
+    {
+      "name": "Piazzolla",
+      "fontFamily": "Piazzolla, serif",
+      "slug": "piazzolla",
+      "category": "serif",
+      "fontFace": [
+        {
+          "src": "http://fonts.gstatic.com/s/piazzolla/v33/N0b52SlTPu5rIkWIZjVKKtYtfxYqZ4RJBFzFfYUjkSDdlqZgy7LYxnLy1AHfAAy5.ttf",
+          "fontWeight": "400",
+          "fontStyle": "normal",
+          "fontFamily": "Piazzolla"
+        },
+        {
+          "src": "http://fonts.gstatic.com/s/piazzolla/v33/N0b72SlTPu5rIkWIZjVgI-TckS03oGpPETyEJ88Rbvi0_TzOzKcQhZqx3gX9BRy5m5M.ttf",
+          "fontWeight": "400",
+          "fontStyle": "italic",
+          "fontFamily": "Piazzolla"
+        },
+      ]
+    }
+  ],
+  "categories": [
+    "sans-serif",
+    "serif"
+  ]
+}
+```
+
+### Create a plugin to provide my font collection
+
+Creating a plugin to provide a font collection can be as simple as this:
+
+```php
+
+<?php
+/*
+Plugin Name: My Font Collection
+Plugin URI: https://your-website.com/
+Description: Add a font collection to your WordPress Font Library.
+Version: 1.0
+Author: Your Name
+Author URI: https://your-website.com/
+License: GPLv2 or later
+Text Domain: my-font-collection
+*/
+
+$my_config = array (
+    'id'             => 'my-font-collection',
+    'name'           => 'My Font Collection',
+    'description'    => 'Demo about how to a font collection to your WordPress Font Library.',
+    'data_json_file' => path_join( __DIR__, 'my-font-collection-data.json' ),
+);
+
+if ( function_exists( 'wp_register_font_collection' ) ) {
+    wp_register_font_collection ( $my_config );
+}
+
+?>
+
+```
+
+### API endpoints
+
+#### Install font families
+Install a font family is to save a font definition in WordPress and make it available to be activated and used by the user.
+
+Save a font family may or may not include storing the font files assets localy to disk in `/wp-content/fonts` folder. That depends on how the font family sent to this endpoint is defined.
+
+```
+POST /wp-json/wp/v2/fonts
+```
+
+Example:
+
+#### Uninstall font families
+Uninstall a font family is to remove the font definition from WordPress and make it unavailable for users to activate and save.
```

Comment:
This is a bit hard to read through. I would rephrase to "Uninstalling a font family removes the font definition from WordPress and makes it unavailable for users to activate and save."